dashboard/client/src/pages/job/JobDetail.tsx

```diff
@@ -162,7 +161,7 @@ export const JobDetailChartsPage = () => {
       >
         <Section>
           <JobProgressBar
-            jobId={jobId}
+            jobId={job.job_id ? job.job_id : undefined}
```

Comment:
Why do we use two` job.job_id` here?

Reply:
what do you mean use two? I'm using a ternary operator to only pass job.job_id if it's not null and pass undefined otherwise.

Reply:
Oops, my mistake haha! I initially thought it was jobId={job.job_id ?? job.job_id : undefined}